docs/content/en/docs/configuration/command-line.md

```diff
@@ -152,6 +195,21 @@ Disables in memory pod list and also pod watch for changes. Pod list and watch i
 
 ---
 
+## --election-id
+
+The ID to be used for electing ingress controller leader.  Defaults to `ingress-controller-leader`.
```

Comment:
Should we mention where this ID is used / stored?

Reply:
This will in the end create a configmap that stores an internal state used to know who is the current leader and if it's alive. The id is used to name this configmap, and the leader is used to update ingress states. I'd leave it as it is (which is different from the previous version of the -h output?) because we already have a new refactor which will reuse this leader elector to the acme functionality, so we'll have a new chance to improve this description.

Reply:
> I'd leave it as it is (which is different from the previous version of the -h output?)

Yeah, in a few cases i've tried to editorialise a little bit, but if you prefer, i'm happy to put the CLI help text verbatim here.